modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+require 'claims_api/claim_logger'
+
+module ClaimsApi
+  class EwsUpdater
+    include Sidekiq::Worker
+
+    def perform(bgs_claim, target_veteran, filed_5103_waiver_ind)
+      bgs_service = BGS::Services.new(
+        external_uid: target_veteran[:external_uid],
+        external_key: target_veteran[:external_key]
+      )
+      bgs_claim[:bnft_claim_dto][:filed_5103_waiver_ind] = filed_5103_waiver_ind
```

Comment:
since we're passing this in it's manipulated state from the controller, we won't need this change done here, as well, correct?

Also, typo again.
`filed5103_waiver_ind`